keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/Capacity.kt

```diff
@@ -20,5 +20,5 @@ package com.netflix.spinnaker.keel.api
 data class Capacity(
   val min: Int,
   val max: Int,
-  val desired: Int
+  val desired: Int? = null
```

Comment:
should there be some validation on the cluster spec to control when this is allowed to be set to null? Like, can this only be null if the scaling policy is not null?

Reply:
ha ha this is below, ignore me :)